code/modules/mechanics/MechanicMadness.dm

```diff
@@ -1388,6 +1388,7 @@
 	proc/setregex(var/datum/mechanicsMessage/input)
 		if(level == 2) return
 		expression = input.signal
```

Comment:
isn't this ignoring flags? shouldn't this be `("[input.signal]/[expressionflag]")`

Reply:
you right